requirements.txt

```diff
@@ -22,6 +22,7 @@ django-jsonview
 django-leaflet
 django-memoize
 django-redis-cache
+django-settings-export
```

Comment:
I used this to access some settings from within the template. Not sure if there is a better way?